packages/nextjs/components/scaffold-eth/Contract/WriteOnlyFunctionForm.tsx

```diff
@@ -93,6 +95,21 @@ export const WriteOnlyFunctionForm = ({
   });
   const zeroInputs = inputs.length === 0 && abiFunction.stateMutability !== "payable";
 
+  const handleCopyCalldata = async () => {
+    try {
+      const calldata = encodeFunctionData({
+        abi: abi,
+        functionName: abiFunction.name,
+        args: getParsedContractFunctionArgs(form),
+      });
+      await navigator.clipboard.writeText(calldata);
+      notification.success("Calldata copied to clipboard!");
+    } catch (e) {
+      console.error("Error copying calldata:", e);
+      notification.error("Failed to copy calldata");
+    }
```

Comment:
Can you update this handling like this: 

```ts
      const errorMessage = getParsedError(e);
      console.error("Error copying calldata:", e);
      notification.error(errorMessage);
```

The `getParsedError` comes from `import { getParsedError, notification } from "~~/utils/scaffold-eth";`

So that if the user doesn't fill proper input data we show more nice info that what's missing.